src/euphorie/content/risk.py

```diff
@@ -660,7 +659,6 @@ def fixed_priority(self, value):
         self.default_priority = value
 
     @property
-    @memoize
```

Comment:
I'm fine with using memoize sparsely and keep this change, but what problem does this solve?
Seems unrelated to the Plone 6 changes here.